plugins/shared-react/package.json

```
@@ -0,0 +1,64 @@
+{
+  "name": "@janus-idp/plugin-frontend-shared-react",
+  "version": "0.1.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
```

```suggestion
"publishConfig": {
    "access": "public",
    "main": "dist/index.cjs.js",
    "module": "dist/index.esm.js",
    "types": "dist/index.d.ts"
  },
```

in other plugins i see as above